src/coreclr/jit/lower.cpp

```diff
@@ -5447,6 +5452,62 @@ GenTree* Lowering::LowerAdd(GenTreeOp* node)
             JITDUMP("Remove [%06u], [%06u]\n", op2->gtTreeID, node->gtTreeID);
             return next;
         }
+#ifdef TARGET_ARM64
+        if (BlockRange().TryGetUse(node, &use))
```

Comment:
What do you need the use for?

Reply:
I thought I had an understanding of this, but I think I actually don't.

I removed this piece from the change.